Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/Home.kt

```diff
@@ -106,7 +106,6 @@ import androidx.compose.ui.unit.dp
 import androidx.hilt.navigation.compose.hiltViewModel
 import androidx.lifecycle.compose.collectAsStateWithLifecycle
 import androidx.window.core.layout.WindowHeightSizeClass
-import androidx.window.core.layout.WindowWidthSizeClass
```

Comment:
I am not sure the background for the import statement removal, but it looks slightly strange for me that symbols are written in fully qualified name.

Reply:
Yeah the codebase currently references 2 different WindowSizeClasses. @jdkoren will help address the ambiguity in a separate PR. Wanted to get this merged in so that Jetcaster is in a buildable state.